p5js/PoseNet/sketch.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2018 ml5
+// 
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ML5 Example
+PoseNet using p5.js
+=== */
+
+let w = 315;
+let h = 315;
+let video;
+let poseNet;
+let poses = [];
+let skeletons = [];
+
+function setup() {
+  createCanvas(w, h);
+  video = createCapture(VIDEO);
+  
+  // Create a new poseNet method with a single detection
+  poseNet = new ml5.PoseNet(video, 'single', gotPoses);
+  
+  // Hide the video element, and just show the canvas
+  video.hide();
+  fill(255, 0, 0);
+  stroke(255, 0, 0);
+}
+
+function draw() {
+  image(video, 0, 0, w, h);
+
+  // We can call both functions to draw all keypoints and the skeletons
+  drawKeypoints();
+  drawSkeleton();
+}
+
+// A function to draw ellipses over the detected keypoints
+function drawKeypoints() {
+  poses.forEach(pose => {
```

Comment:
I've started using this style/syntax more now myself, but I think for the p5 examples  we should probably stick with more vanilla for loops or `for of` loops.

Reply:
Yes, agree. Will update this.